web/src/pages/Settings/Settings.jsx

```diff
@@ -77,102 +94,85 @@ const Settings = ({ openPage }) => {
-                            >
-                                <div className="groups-icon">
-                                    <SpeakerGroupIcon fontSize="inherit" />
-                                </div>
-                            </ListItem>
-                        </>
-                    )}
-
-                    {/* <ListItem
-            name="Sessions"
-            onClick={() => router.navigate("/settings/sessions")}
-          >
-            <div className="sessions-icon"><CableIcon fontSize="inherit"/></div>
-          </ListItem> */}
-
-                    <ListItem
-                        name="Presets"
-                        onClick={() => router.navigate("/settings/presets")}
-                    >
-                        <div className="presets-icon">
-                            <PlaylistAddIcon fontSize="inherit" />
-                        </div>
-                    </ListItem>
-
-                    <ListItem
-                        name="Configuration"
-                        onClick={() => router.navigate("/settings/config")}
-                    >
-                        <div className="config-icon">
-                            <HandymanIcon fontSize="inherit" />
-                        </div>
-                    </ListItem>
-
-                    <ListItem
-                        name="Updates"
-                        onClick={() => {
-                            window.location.href =
-                                "http://" + window.location.hostname + ":5001/update";
-                        }}
-                    >
-                        <div className="update-icon">
-                            <UpdateIcon fontSize="inherit" />
-                        </div>
-                    </ListItem>
-
-                    <ListItem
-                        name="About"
-                        onClick={() => router.navigate("/settings/about")}
-                    >
-                        <div className="about-icon">
-                            <InfoIcon fontSize="inherit" />
-                        </div>
-                    </ListItem>
-                    {IsMobileApp() && (
-                        <div>
-                            <text>Always connect to this Amplipi</text>
-                            <Checkbox
-                                checked={isSavedUrl}
-                                onChange={(e) => {
-                                    if (e.target.checked) {
-                                        SaveURL();
-                                    } else {
-                                        UnsaveURL();
-                                    }
-                                    setIsSavedUrl(e.target.checked);
-                                }}
-                            />
-                        </div>
-                    )}
-                </List>
-            </div>
+            <List style={{width: "400px", maxWidth: "100%"}}>
```

Comment:
This inline styling is an approximation of the previous styling, I used an approximation as I couldn't find what was making the old list be as wide as it was when the new one wasn't like that by default

It looks the exact same on my computer screen, but will never be too large to fit the screen either. Should be the same on mobile, too; testing shortly